site/en/blog/new-in-devtools-112/index.md

```diff
@@ -0,0 +1,237 @@
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/f5266ee227449dbbc3bc599df1b38cdb36cae4cb #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/d4748c98971bfff697f209fe11de892a5b93aca6 #}
+
+Chromium issue: [1401107](https://crbug.com/1401107).
+
+### CSS nesting support {: #nesting }
+
+The **Elements** > **Styles** pane now recognizes [CSS Nesting](https://w3c.github.io/csswg-drafts/css-nesting/) syntax and applies nested styles to the right elements.
+
+{% Video src="video/NJdAV9UgKuN8AhoaPBquL7giZQo1/Wog2uOaJTV84OtXcHpYH.mp4", autoplay="true", muted="true", loop="true", controls="true", class="screenshot" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/f1ed9b6180cb75fcfd43dfac95ac9a40c35e03df #}
+
+Chromium issue: [1172985](https://crbug.com/1172985).
+
+## Logpoint and conditional breakpoint improvements {: #logpoint }
+
+Further improving the [enhanced breakpoint UX](/blog/new-in-devtools-111/#breakpoint-redesign), the **Console** now marks messages triggered by breakpoints:
+
+- `console.*` calls in [conditional breakpoints](/docs/devtools/javascript/breakpoints/#conditional-loc) with an orange question mark `?`
+- [Logpoint](/docs/devtools/javascript/breakpoints/#log-loc) messages with pink two dots `..`
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/5udIX9W4LFcDb3H6DuDp.png", alt="Changes to how the Console now displays messages triggered by breakpoints: with icons and proper source link.", width="800", height="566" %}
+
+The **Console** now gives you proper anchor links to breakpoints in source files instead of `VM<number>` scripts that Chrome creates to run any piece of Javascript on [V8](https://v8.dev/).
+
+Click the link next to the breakpoint message to jump directly to the breakpoint editor.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/8lAz0lb168HXKvhscP2Q.png", alt="The anchor link next to a logpoint message that opens the breakpoint editor.", width="800", height="811" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/c845a441b0fe05c22f88cdb23463edee2b5985b7 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/9762db476cd7414d3ce351f32a0564421f66901f #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/42448cc63567ac407fd2088597da83aff17c5b55 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/4739f48e50d41025aba3c2af94e61cc3069aa563 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/bb0e41ed3c30bd988c49a76f0cf084f58c0bddc2 #}
+
+Chromium issue: [1027458](https://crbug.com/1027458).
+
+## Ignore irrelevant scripts during debugging {: #ignore-list }
+
+To help you focus on the most important parts of your code, you can now add irrelevant scripts to the **Ignore List** right from the file tree on the **Sources** > **Page** pane. The debugger ignores all scripts added to the list. 
+
+Right-click any script or folder and select one of the ignore-related options.
```

Comment:
```suggestion
Right-click any script or folder and select one of the ignore-related options.
```
Explain what happen after we ignore a script. It reduce the frame in stacktrace and call stack, we hide frames that are ignored. We might have a docs already. Link to that. 

Below is the relevant previous WNDT.
https://developer.chrome.com/blog/new-in-devtools-106/#call-stack
https://developer.chrome.com/blog/new-in-devtools-106/#auto-ignore